tests/syngenta_digital_dta/s3/test_s3.py

```diff
@@ -49,7 +49,8 @@ def test_create(self):
         results = self.adapter.create(
             s3_path='test/test-create.json',
             data={'test': True},
-            json=True
+            json=True,
+            tags='partener=ingestor&integrationAccountReference=dummy_account'
         )
         self.assertEqual(results['ResponseMetadata']['HTTPStatusCode'], 200)
```

Comment:
you should create a separate unit test for creating object with tagging, assertion of this new unit test should retrieve the object after creation and check if the tags have been assigned to the object

Reply:
@abdullah-syngenta can tags be provided by user as a dictionary?

Reply:
@ncgl-syngenta we already modified the code to takes the tags as dict of key value pairs